Sources/Mantis/Helpers/LocalizedHelper.swift

```diff
@@ -9,13 +9,23 @@
 import Foundation
 
 struct LocalizedHelper {
-    static func getString(_ key: String, value: String? = nil, comment: String = "") -> String {
+    static func getString(
+        localizationConfig: LocalizationConfig,
```

Comment:
Can you set Mantis.localizationConfig as the default value of localizationConfig? Then we don't need to pass Mantis.localizationConfig every time when using it.
```Swift
static func getString(
         localizationConfig: LocalizationConfig = Mantis.localizationConfig,
         key: String,
         value: String? = nil
     ) -> String
```

Reply:
👍 will do, great idea